test/modules/standard/Random/bradc/testGetNth.chpl

```diff
@@ -13,7 +16,8 @@ for i in 1..n {
   if (r1 != r2 || r2 != r3) {
     writeln("mismatch at #", i, ": ", r1, " ?= ", r2, " ?= ", r3);
   } else {
-    writeln("#", i, " = ", r2);
+    // Don't print the #s in this test; just check for a match
+    //writeln("#", i, " = ", r2);
```

Comment:
Doesn't this weaken the test in that -- if someone broke the RandomStream implementation such that it always returned 0.0, this test would now pass?  The intention of the old .good was to lock expected values into a test to help avoid the chances of someone systematically breaking the stream itself and to hard-code it as generating "what NPB expects for a given seed value).